README.md

```diff
@@ -12,3 +12,42 @@ A pretty simple command-line realtime chat application built using Flask, Socket
 
 ## 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/prettyirrelevant/cli-chat)
+
+## Installation and Use
+
+To begin installation clone the repo into preferred directory
+```
+git clone https://github.com/Johnkayode/Dominos-bot.git
+```
+Activate a virtual environment (not necessary)
```

Comment:
using a virtual environment is important as to note pollute global environment.

Reply:
Done